CloudFormation/pint/sles12sp2.templ

```diff
@@ -0,0 +1,132 @@
+{
+  "AWSTemplateFormatVersion" : "2010-09-09",
+
+  "Description" : "SLES12 SP2 on-demand auto AMI ID",
+
+  "Parameters" : {
+    "KeyName" : {
+      "Description" : "Name of an existing EC2 KeyPair to enable SSH access to the instances",
+      "Type" : "AWS::EC2::KeyPair::KeyName",
+      "ConstraintDescription" : "must be the name of an existing EC2 KeyPair."
+    },
+    "SecurityGroups" : {
+      "Description" : "EC2 SecurityGroups for SUSE Manager instance",
+      "Type" : "List<AWS::EC2::SecurityGroup::GroupName>",
+      "ConstraintDescription" : "must be an existing SecurityGroup"
+    },
+    "ServerInstanceType" : {
+      "Description" : "Server EC2 instance type",
+      "Type" : "String",
+      "Default" : "t2.micro",
+      "ConstraintDescription" : "must be a valid EC2 instance type"
+    }
+  },
+
+  "Resources" : {
+    "SLES12Instance" : {
+      "Type" : "AWS::EC2::Instance",
+      "Properties" : {
+        "InstanceType"   : { "Ref" : "ServerInstanceType" },
+        "SecurityGroups" : { "Ref" : "SecurityGroups" },
+        "KeyName" : { "Ref" : "KeyName" },
+        "ImageId": { "Fn::GetAtt": [ "AMIInfo", "Id" ] }
+      }
+    },
+  
+    "AMIInfoFunction": {
+      "Type": "AWS::Lambda::Function",
+      "Properties": {
+        "Code": {
+	  "ZipFile": { "Fn::Join": ["\n", [
+	    "from botocore.vendored import requests",
+	    "import json",
+            "import cfnresponse",
+            "import sys",
+            "import re",
```

Comment:
I didn't see your comments before. There was nothing in the pull requests anyway.

This one is the good old tabs vs spaces. I'll convert them into spaces.